src/components/NcActionRadio/NcActionRadio.vue

```diff
@@ -157,7 +158,11 @@ export default {
 		'change',
 	],
 
-	setup() {
+	setup(props) {
+		if (typeof props.modelValue === 'boolean') {
+			Vue.util.warn('[NcActionRadio] Boolean type of `modelValue` is deprecated and will be removed in next versions')
```

Comment:
This will be only shown if set initially.
Meaning if the prop is mutated to a boolean it is not shown again